RELEASE_NOTES.md

```diff
@@ -1,8 +1,13 @@
 # Release Notes
 
-## 5.20.4-alpha - tbd
-
-* tbd
+## 5.20.4-alpha - 2021-03-31
```

Comment:
Thanks,

maybe even remove the -alpha to go for a 'real' release :)

Basically - at least that's what I ususally did, we can change that, there might be a better way -
- When I want to to a release I make a single commit to remove the "-alpha" (prerelease flag)
- Build & Release 
- Rename the "Next Release" to "Release <Version>" PR and merge it to master
- Make a new commit to add a new section with the prerelease flag
- Start a new "Next Release" PR

Like said above we can change that workflow if you have other ideas.

Reply:
I removed the -alpha flag.
I'm ok with your release flow, thanks for explaining it.